sickbeard/providers/generic.py

```diff
@@ -384,7 +384,7 @@ def findSearchResults(self, show, episodes, search_mode, manualSearch=False, dow
 
             # parse the file name
             try:
-                myParser = NameParser(False)
+                myParser = NameParser(showObj=show)
```

Comment:
This is good, we should always pass the show object to NameParser if we have it.